corehq/apps/hqwebapp/static/hqwebapp/js/main.js

```diff
@@ -325,7 +325,7 @@ var COMMCAREHQ = (function () {
 function setUpIeWarning() {
     'use strict';
     var $warning;
-    if ($.browser.msie) {
+    if (/msie/i.test(navigator.userAgent) || /windows/i.test(navigator.userAgent)) {
```

Comment:
> no B3 templates to my knowledge are using main.js

As of this PR, the web users page is using main.js. Needed it for undoing deleted users, which hadn't been working on the B3 version of the page.